Gemfile

```diff
@@ -8,6 +8,7 @@ ruby File.read(".ruby-version").strip
 gem "rails", "6.1.3.2"
 
 gem "bootsnap"
+gem "dalli"
```

Comment:
What is this used for? It'd be good to have an explanation in the commit message.